src/components/EngagementDataGrid/EngagementColumns.tsx

```diff
@@ -29,31 +28,68 @@ import {
   useEnumListFilterToggle,
   useFilterToggle,
 } from '../Grid';
+import { AssignmentIcon } from '../Icons';
```

Comment:
```suggestion
import { AssignmentIcon } from '@mui/icons-material';
```

Though I don't really think that express's this intent very well.
`OpenInNew` or `Link` I think are better fits. Thoughts?

Reply:
OpenInNew works if this should be creating a new tab, which I think is reasonable for this.

Reply:
Well I don't want it opening a new tab by default. But I hear you that's where we usually see this icon.
I'm looking for a generic "go to" icon.
That way we can consistently using this icon in multiple grids. Like the progress report one.

Reply:
After looking through the icons, I used the Link icon as it is probably the must intuitive and appropriate.